src/sass/partials/layouts/_events.scss

```diff
@@ -62,6 +62,7 @@
         font-size: 90%;
       }
       .card-date-text {
+        width:100%;
```

Comment:
You need to center align the text if you are gonna make the width 100%. Currently it stays in the left side.